tests/resources/cfg.yml

```diff
@@ -69,7 +69,7 @@ order:
 # Line Items (required)
 # Allowed Templating Key Words: Bidder, Media, CPM, Misc
 #
-# Supported Types: price_priority, standard
+# Supported Types: price_priority, standard, sponsorship
```

Comment:
This should be added here as well:
https://github.com/prebid/line-item-manager/blob/master/line_item_manager/conf.d/line_item_manager.yml#L93